src-tauri/tests/fixtures/ftp/slowftp.py

```
@@ -0,0 +1,584 @@
+--late-final is the part that makes the third case possible, and it is the whole
+point. When our client gives up reading, this server does NOT close and does NOT
+cancel: it finishes its work and sends the closing 226 late, on the same control
+connection. That reply is then sitting in the socket when the NEXT command asks
+its own question, and the next command reads it as its own answer. A fixture that
+tears down at the deadline cannot produce this: the green it returns would only
+be saying "the fixture closed", which is the earlier trap wearing a new face.
+"""
+import argparse, os, select, socket, ssl, threading, time, sys
+
+def log(msg):
+    sys.stderr.write("[slowftp] %s\n" % msg); sys.stderr.flush()
```

Ruff (0.16.2)

[error] Multiple statements on one line (semicolon)

(E702)